department-of-veterans-affairs/caseflow
# Clear assignees on cancel and submit

Resolves #14213

### Description
Clears assignee selections in state when cancelling or on a successful submit.

### Acceptance Criteria
- [x] Assignee selections are cleared when cancelling and reentering assign modal
- [x] Assignee selections are cleared when submitting and reentering assign modal

### Testing Plan
1. Sign in a BVAAABSHIRE and go to any case with an open judge assign task
1. Select "Assign to Attorney" in dropdown
1. Select any attorney for you team and hit cancel
1. Without refreshing, select "Assign to Attorney" in dropdown
    - [x] Confirm attorney selection is cleared
1. Repeat the above by selecting "Other", cancel, and reselect "Assign to Attorney"
    - [x] Confirm "other" selection is cleared
1. Repeat the above by selecting "Other", selecting an attorney in the new dropdown, cancel, and reselect "Assign to Attorney"
    - [x] Confirm "Other" and attorney selection are cleared
1. Select "other", an attorney, enter some mock instructions and hit submit.
1. Without refreshing, hit back to get to case details, select "Assign to Attorney" in dropdown
    - [x] Confirm all dropdown selections are cleared. (Note your instructions will still show)

### User Facing Changes
 - [x] Attorney selections are not persisted when cancelling or on a successful submit

 BEFORE|AFTER
 ---|---
![Screen Shot 2020-06-01 at 8 47 55 AM](https://user-images.githubusercontent.com/45575454/83410579-9bcd1080-a3e4-11ea-81e6-bd43ca38b88e.png)|![Screen Shot 2020-06-01 at 8 47 46 AM](https://user-images.githubusercontent.com/45575454/83410588-9d96d400-a3e4-11ea-93e0-76296d12ba50.png)
